Vineyard/Vineyard/ContentView.swift

```diff
@@ -12,7 +12,18 @@ struct ContentView: View {
     var body: some View {
         NavigationStack {
             VStack {
-                Text("\(viewModel.user.name)")
+                HStack {
```

Comment:
I would recommend switching this from an HStack to a toolbar. This is a good site which holds all that info: https://holyswift.app/how-to-create-toolbar-in-swiftui/.

Reply:
Replaced. Is that what you were looking for?